### ISSUE TITLE
Hardiness / Heat Zone / Growing degree days

### ISSUE THREAD

**Issue by andru** [openfarmcc/OpenFarm#59]
I saw USDA hardiness and AHS heat zones mentioned in the Guide wireframe so I thought I'd bring it up for discussion.

_[USDA hardiness zones](https://en.wikipedia.org/wiki/Hardiness_zone)_ are a mostly U.S. specific metric (some other systems are in use to model climate factors in other countries like Australia and Canada) based on minimum winter temperatures and give a very approximate indication of the ability for a plant to _survive_ in a given climate. It does not indicate a plant will prosper and because it only measures minimum winter temperatures, locations with similar winter temperatures but different summer temperatures will be given the same hardiness zone. Nonetheless it's a useful indicator when used appropriately especially for perennial plants like fruit trees. For the most part, it's irrelevant for annual crops where summer temperatures are usually the most important metric.

_[AHS heat zones](https://en.wikipedia.org/wiki/Hardiness_zone#AHS_heat_zones)_ give the average number of days a particular location experiences temperatures of over 30ºC. It's a proprietary system with, to my experience, almost no practically useful data available. Most flowering plants do not mature any faster over 30ºC, so I'm not sure why this metric exists.

_[Growing degree days](https://en.wikipedia.org/wiki/Growing_degree_day)_ are a cumulative tally of hours of average daily temperatures.  They can be used very reliably (and is used in industrial agriculture) to indicate when a crop will flower, when fruit will mature, etc. There is a decent amount of data available for various species and specific varieties commonly grown in industrial agriculture, but not much data available for less common garden crops. It's relevant to growers in both indoor/controlled and outdoor environments.

In short, I think OpenFarm should definitely include USDA hardiness zones where possible, with some way to clearly indicate it's only relevant in regards to minimum temperatures, and absolutely include a growing degree days value where data is available, ideally with a way to educate people on this system and provide them with information on the average growing degree days in their area, since it's mostly unknown to hobby/small gardeners.

**Comment by simonv3** [comment]
Turns out these weren't editable yet. I just did this in 433bed57fd502a2f7dcefdf771da62b969f364d8
